+ [**Ping Zeng**](https://github.com/biostatpzeng), Email: zpstat@xzhmu.edu.cn
+ ![#f03c15](https://placehold.it/15/f03c15/000000?text=+) Master Students
+ #: Corresponding Authors; $: Joint First Authors
+ *The research is supported in part by NFSC and other fundings from Chinese Government.*
:blush:  

----------------------------------------------------------------------------------------------------------------------------------------
> ## In preparation
----------------------------------------------------------------------------------------------------------------------------------------
> ## 个人情况
+ （1）曾平，博士，硕士生、博士生导师；徐州医科大学公共卫生学院副院长，任职卫生统计学教研室，流行病与卫生统计学教授，公共卫生与预防医学一级学科负责人、公共卫生专业学科负责人、生物统计学二级博士点学科负责人；
+ （2）2012年9月至2015年7月于南京医科大学公共卫生学院学习，获得流行病与卫生统计学博士；博士毕业论文受到南京医科大学优秀博士论文培育项目和江苏省教育厅普通高校研究生科研创新计划项目支持。2015年10月-2017年10月在美国密歇根大学生物统计学系从事博士后研究（NIH项目支持）；
+ （3）中国卫生信息与医疗健康大数据学会统计理论与方法专业委员会常务委员；
+ （4）2018年获徐州医科大学党委“青年教学科技人才”称号、江苏省高校“青蓝工程”优秀青年骨干教师称号、徐州医科大学“最有范导师”；2019年获徐州医科大学第一届“十大杰出青年人才”称号、江苏省“六大人才高峰”高层次人才称号；2022年获江苏省高校“青蓝工程”中青年学术带头人称号；2024年获徐州医科大学第三届“优秀导师”称号；
+ （5）是中国卫生统计、Genome Biology、Nature Human Behavior、Nature Biomedical Engineering、Nature Communications、Cell Genomics、Bioinformatics、BMC Medicine、Briefings in Bioinformatics、Nucleic Acids Research、Journal of Applied Statistics、Computational Statistics and Data Analysis、Statistics in Medicine、Heredity、Linear Algebra and its Applications、Communications biology、Computers in Biology and Medicine、Environmental Advances、BMC Genomic Data、BMC Genomics、iScience、Journal of Translational Medicine、Environmental Research、Annals of Human Genetics、Genetic Epidemiology、International Journal of Oncology、Statistica Neerlandica、Neural Regeneration Research等期刊的审稿人；
+ （6）徐州市卫健委、国家自然科学基金、多个省自然科学基金、英国医学研究委员会（Medical Research Council, MRC）、英国研究与创新署（UK Research and Innovation, UKRI）的评审专家。

> ## 研究方向
+ （1）基于大型队列人群数据，利用各种流行病学策略（如因果推断、机器学习）和现代科技手段（如空间遥感技术），同时发展统计建模和计算方法，从群体角度了解暴露（如空气污染、生命早期暴露、不良社会行为心理特征）对后期疾病（如神经退化、神经精神、肿瘤、免疫、代谢和心脑血管等疾病）发生发展的影响；
+ （2）聚焦复杂疾病（如神经退化、神经精神、肿瘤、免疫、代谢和心脑血管等疾病），利用生物医学大数据发展多组学整合、遗传关联、风险预测、高维中介和因果推断等统计方法，从表型组、基因组、表观组、转录组、蛋白组和代谢组等多组学视角了解疾病发生发展过程，为疾病诊断和预后提供科学依据和参考。

> ## 研究成果
+ （1）发表英文论文近50篇、发表中文核心论文近40篇（https://github.com/biostatpzeng/pingzeng/blob/master/publications/English%20publications.md）；
+ （2）关于统计遗传学方法与理论成果发表在PLoS Computational Biology、Briefings in Bioinformatics、Nature Communications、Bioinformatics、PLoS Genetics、Human Genetics、Computational and Structural Biotechnology Journal、Human Molecular Genetics、Current Genetics、BMC Genomics、Journal of Applied Statistics、BMC Bioinformatics、BMC Medical Research Methodology和Annals of Human Genetics等统计和遗传期刊；
+ （3）关于数据分析和理论方法应用的成果发表在中国卫生统计、International Journal of Epidemiology、Annals of Neurology、BMC Medicine、Human Molecular Genetics、Journal of Clinical Endocrinology & Metabolism、Journal of Bone and Mineral Research、Neurobiology of Aging、Progress in Neuro-psychopharmacology and Biological Psychiatry、Journal of Translational Medicine等遗传期刊。
+ （4）徐州市自然科学优秀学术论文二等奖（2018）、教育部高等学校科学研究优秀成果二等奖（2019）、卫健委医学新技术引进一等奖（2020）、江苏省高等学校科学技术研究成果奖三等奖（2021）。

> ## 科研条件
+ （1）拥有闵行出生队列、UKB生物银行、CHNS、CHARLS、CLHLS、GERA和WTCCC等数十个大型GWAS个体水平数据（individual-level data）；拥有包括神经退化性疾病、神经精神疾病、免疫、代谢和心脑血管疾病在内的数千个复杂性状汇总水平数据（summary-level data）；能够用于统计遗传和队列人群高维数据建模和实际分析；
+ （2）徐州医科大学医学统计分析中心；
+ （3）徐州医科大学校重点实验室人类遗传和环境医学重点实验室；
+ （4）徐州医科大学高性能计算（HPC）集群中心；
+ （5）徐州市生物数据挖掘与医疗健康转化创新工程研究中心；
+ （6）江苏省生物数据挖掘与医疗健康转化工程研究中心。

> ## 科研项目
+ 2018年至今，主持科研课题包括：
+ （14）国家自然科学基金面上项目（82574203）（2026.01~2029.12）
+ （13）江苏省卫生应急研究所、徐州医科大学医疗卫生应急救援研究中心招标课题（JSWSYJ-20240203）（2025.01~2026.12）
+ （12）国家卫生健康委避孕药具警戒与生育力监测重点实验室面上项目（JSHD202412）（2025.01~2026.12）
+ （11）煤炭环境致病与防治教育部重点实验室开放课题基金重点项目（MEKLCEPP/SXMU-202415）（2024.09~2026.08）
+ （10）江苏省自然科学基金面上项目（BK20241952）（2024.09~2027.08）
+ （9）国家自然科学基金面上项目（82173630）（2022.01~2025.12）
+ （8）江苏省高校青蓝工程中青年学术带头人项目（2022.07~2025.07）
+ （7）徐州医科大学青年科技创新团队项目（TD202008）（2020.09~2024.09）
+ （6）中国博士后科学基金特别项目（2019T120465）（2019.08~2024.01）
+ （5）江苏省六大人才高峰项目（WSN-087）（2019.08~2024.10）
+ （4）中国博士后科学基金一等项目（2018M630607）（2018.08~2024.1）
+ （3）江苏省自然科学基金面上项目（BK20181472）（2018.08~2021.08）
+ （2）教育部人文社会科学研究青年基金项目（18YJC910002）（2018.08~2021.08）
+ （1）江苏省高校青蓝工程优秀青年骨干教师项目（2018.08~2021.08）

> ## 研究生培养
+ （1）公共卫生专业2020级研究生张萌完成的《从遗传相关性、中介和因果关系的角度重新审视出生体重与乳腺癌之间的关系》论文被评为2024年江苏省优秀专业学位硕士学位论文；
+ （2）公共卫生专业2019级研究生陆皓杰完成的《整合eQTL信息的多位点单核苷酸多态性等级Cox生存关联分析方法》论文被评为2023年江苏省优秀专业学位硕士学位论文；
+ （3）流行病与卫生统计学专业2017级研究生余星皓完成的《建立基于模型平均方法的全基因组表型预测模型》论文被评为2021年江苏省优秀学术学位硕士学位论文。

> ## 统计咨询
+ （1）面向临床一线研究人员及医学生开展针对临床研究的统计方法和大数据分析咨询与服务；
+ （2）帮助临床科研人员解决面临的各种统计学问题。

+
+

+ Hao Zhang<sup>$</sup>, Yuxin Liu, Yu Yan, Jike Qi, Hua Lin, Yuchen Jiang, Xinyi Wang, Hongyan Cao, Zhou Jiang, Shuo Zhang, Ting Wang, Yue Xu, Weiyi Song, Ke Wang<sup>#</sup>, Chu Zheng<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)# (2025). Unraveling the role of inflammatory markers in coronary artery disease risk via association, mediation and prediction analyses.

+ Xin Zhang<sup>$</sup>, Shuo Zhang<sup>$</sup>, Zixin Wang, Chu Zheng<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)# (2025). Mediation effects of inflammation on the association of physical activity and chronic kidney disease: evidence from questionnaire and device-measured assessments.

+ Shuo Zhang<sup>$</sup>, Jike Qi<sup>$</sup>, Yuchen Jiang<sup>$</sup>, Hua Lin, Xinyi Wang, Ting Wang, Hongyan Cao and [**Ping Zeng**](https://github.com/biostatpzeng)# (2025). An integrative association analysis for complex diseases in underrepresented groups by leveraging the trans-ethnic genetic similarity.

+ Yuchen Jiang<sup>$</sup>, Jike Qi<sup>$</sup>,Hua Lin, Xinyi Wang, Yuxin Liu, Yu Yan, Ting Wang, Jingjing Sun, Min Sun and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2025). Evaluating the link between metabolic syndrome score and 47 immune diseases from a multidimensional perspective.

+ Zhou Jiang<sup>$</sup>, Yu Yan<sup>$</sup>, Jike Qi<sup>$</sup>, Guojing Yuan, Hao Zhang, Yuxin Liu and [**Ping Zeng**](https://github.com/biostatpzeng)# (2025). Proteomic insights into the association between cigarette smoking and MASLD via proteome-wide Mendelian randomization-based mediation analyses.






+ Yu Yan, Yuchen Jiang, Yuxin Liu, Jike Qi, Hua Lin, Xinyi Wang, Hao Zhang, Zhou Jiang,Ke Wang, Yue Xu<sup>#</sup>, Chu Zheng<sup>#</sup>, and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Ocular extraintestinal manifestations of IBD: phenotypic and genetic evidence from prospective cohort study and instrumental variable causal inference.

+ Ying Liu<sup>$</sup>, Guojing Yuan<sup>$</sup>, Zhou Jiang<sup>$</sup>, Yuxin Liu, Yu Yan, Hao Chen, Shuo Zhang, Ting Wang, Ke Wang<sup>#</sup>, Chu Zheng<sup>#</sup>, and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Exploring the influence of overall lung function on incident dementia risk in a prospective national cohort study.

+ Zixin Wang<sup>$</sup>, Hao Zhang<sup>$</sup>, Zhou Jiang, Yuxin Liu, Xin Zhang, Yu Yan, Shuo Zhang, Chu Zheng, Yue Xu<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Exploring the phenotypic and causal relationships between eight female reproductive factors and delayed biological aging in two national population-based studies.

+ Yu Yan<sup>$</sup>, Yuxin Liu<sup>$</sup>, Jike Qi<sup>$</sup>, Keying Chen, Yue Xu, Shuo Zhang, Ke Wang, [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> and Chu Zheng<sup>#</sup> (2024). Disentangling the association of depression and anxiety with migraine via a prospective multi-ancestry cohort study and causal inference analysis.

+ Kexuan Zhu<sup>$</sup>, Xinying Guan<sup>$</sup>, Yu Yan<sup>$</sup>, Keying Chen, Tongyu Gao, Hao Zhang, Ting Wang, Chu Zheng<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). A genome-wide cross-trait analysis identifies shared loci and causal relationships between female-specific health factors and Parkinson's disease.

+ Jike Qi<sup>$</sup>, Xinying Guan<sup>$</sup>, Zhou Jiang, Keying Chen, Hao Zhang, Tongyu Gao and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Unraveling the intricate linkages between kidney function and Parkinson's disease: an integrative exploration at the crossroads of population and genetic perspectives.

+ Yuxin Liu<sup>$</sup>, Wenyan Hou<sup>$</sup>, Tongyu Gao, Yu Yan, Ting Wang, Chu Zheng and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Influences and roles of polygenic risk score in the development and age at onset of 32 complex diseases.

+ Ying Liu<sup>$</sup>, Zhou Jiang<sup>$</sup>, Guojing Yuan<sup>$</sup>, Yuxin Liu, Shuo Zhang, Ting Wang, Chu Zheng<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Modifying role of genetic susceptibility in the associations of accelerated aging and lung function with dementia.

+ Yu Yan, Tongyu Gao, Keying Chen, Yuxin Liu, Hao Zhang, Zhou Jiang, Ting Wang, Yue Xu, Ke Wang, Chu Zheng<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Ocular extraintestinal manifestations of IBD: insights from large-scale genome-wide cross-trait analysis and causal inference.

+ Yuxuan Wu, Yu Yan, Yue Xu, Yuchen Jiang, Ting Wang, [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> and Chu Zheng<sup>#</sup> (2023). Maternal smoking during pregnancy, genetic factors, and offspring depression: a prospective analysis in the UK Biobank study.

+ Tongyu Gao, Yin Liu, Siyi Jin, [**Ping Zeng**](https://github.com/biostatpzeng), Ting Wang and Chunxia Miao<sup>#</sup> (2024). Exploring the maternal origin of children-onset asthma with polygenetic risk score based instrumental variable causal inference method.

+ Tongyu Gao, Zhou Jiang, Keying Chen, Ying Liu, Kexuan Zhu, Hao Zhang, Ting Wang and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). A comprehensive analysis of the association between blood cell traits and myocardial infarction through Mendelian randomization and genome-wide genetic pleiotropy. 

+ Chu Zheng, Tongyu Gao, Jike Qi, Yue Xu, Ke Wang and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Improved prediction accuracy of genetic risk score model for stroke in minority populations via transfer learning methods.

+ Zhou Jiang<sup>$</sup>, Shuo Zhang<sup>$</sup>, Yiyang Zhu, Ying Liu and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2023). Composite null hypothesis Mendelian randomization mediation analysis of proteins from smoking to diseases using GWAS summary statistics.

+ Lei Xia, Hao Zhang, Tongyu Gao, Jike Qi, Yu Yan, Yuxin Liu, Shuiping Huang<sup>$</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2024). Exploring potential relationships of plasma proteins with cardiovascular diseases based on GWAS summary statistics.

+ [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> and Xiang Zhou<sup>#</sup> (2021). Mediation analysis for omics studies using summary statistics.
----------------------------------------------------------------------------------------------------------------------------------------
> ## In preparation
----------------------------------------------------------------------------------------------------------------------------------------






> ## 2021
+ Haojie Lu<sup>$</sup>, Jiahao Qiao<sup>$</sup>, Zhonghe Shao<sup>$</sup>, Ting Wang, Shuiping Huang and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). A comprehensive gene-based pleiotropic association analysis for 14 psychiatric disorders with GWAS summary statistics. **BMC Medicine**, in press.

+ Haojie Lu<sup>$</sup>, Yongyue Wei<sup>$</sup>, Zhou Jiang, Jinhui Zhang, Ting Wang, Shuiping Huang and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). Integrative eQTL-weighted hierarchical Cox models for SNP-set based time-to-event association studies. **Journal of Translational Medicine**, in press.

+ Ting Wang, Haojie Lu and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). Identifying pleiotropic genes for complex phenotypes with summary statistics from a perspective of composite null hypothesis testing. **Briefings in Bioinformatics**, in press.

+ Zhonghe Shao<sup>$</sup>, Ting Wang<sup>$</sup>, Meng Zhang, Zhou Jiang, Shuiping Huang<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). IUSMMT: survival mediation analysis of gene expression with multiple DNA methylation exposures and its application to cancers of TCGA. **PLoS Computational Biology**, 17(8): e1009250. [doi:10.1371/journal.pcbi.1009250](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009250).

+ Haimiao Chen<sup>$</sup>, Jinhui Zhang<sup>$</sup>, Ting Wang<sup>$</sup>, Shuo Zhang, Qingwei Lai, Shuiping Huang<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). Type II diabetes mellitus and amyotrophic lateral sclerosis: genetic overlap, causality, and mediation. **Journal of Clinical Endocrinology & Metabolism**, dgab465. [doi: 10.1210/clinem/dgab465](https://academic.oup.com/jcem/advance-article-abstract/doi/10.1210/clinem/dgab465/6309609?redirectedFrom=fulltext).

+ Haojie Lu<sup>$</sup>, Ting Wang<sup>$</sup>, Jinhui Zhang, Shuo Zhang, Shuiping Huang and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). Evaluating genetic correlation of associated loci for complex diseases and traits between European and East Asian populations. **Human Genetics**, 140(9): 1285-1297. [doi:10.1007/s00439-021-02299-8](https://link.springer.com/article/10.1007%2Fs00439-021-02299-8).

+ [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup>, Zhonghe Shao and Xiang Zhou<sup>#</sup> (2021). Statistical methods for mediation analysis in the era of high-throughput genomics: current successes and future challenges. **Computational and Structural Biotechnology Journal**, 19: 3209-3224. [doi: 10.1016/j.csbj.2021.05.042](https://www.sciencedirect.com/science/article/pii/S2001037021002233?via%3Dihub).

+ Xinghao Yu, Yongyue Wei, [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> and Shufeng Lei<sup>#</sup> (2021). Birth weight is positively associated with adult osteoporosis risk: observational and Mendelian randomization studies. **Journal of Bone and Mineral Research**, 36(8): 1469-1480. [doi: 10.1002/jbmr.4316](https://asbmr.onlinelibrary.wiley.com/doi/10.1002/jbmr.4316).

+ Jinhui Zhang<sup>$</sup>, Haojie Lu<sup>$</sup>, Shuo Zhang, Huashuo Zhao, Ting Wang, Fengjun Guan<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). Leveraging methylation alterations to discover potential causal genes associated with the survival risk of cervical cancer in TCGA through a two-stage inference approach. **Frontiers in Genetics**, 12: 667877. [doi: 10.3389/fgene.2021.667877](https://www.frontiersin.org/articles/10.3389/fgene.2021.667877/full).

+ Yixin Gao<sup>$</sup>, Jinhui Zhang<sup>$</sup>, Huashuo Zhao, Fengjun Guan<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). Instrumental heterogeneity in sex-specific two-sample Mendelian randomization: causal effects of anthropometric traits on breast/prostate cancer. **Frontiers in Genetics**, 12: 651332. [doi: 10.3389/fgene.2021.651332](https://www.frontiersin.org/articles/10.3389/fgene.2021.651332/full).

+ [**Ping Zeng**](https://github.com/biostatpzeng), Jing Dai, Siyi Jin and Xiang Zhou (2021). Aggregating multiple expression prediction models improves the power of transcriptome wide association studies. **Human Molecular Genetics**, 30(10): 939-951. [doi: 10.1093/hmg/ddab056](https://academic.oup.com/hmg/article-abstract/30/10/939/6145951?redirectedFrom=fulltext).

+ Lu Liu, [**Ping Zeng**](https://github.com/biostatpzeng), Fuzhong Xue, Zhongshang Yuan<sup>#</sup> and Xiang Zhou<sup>#</sup> (2021). Multi-trait transcriptome-wide association studies with probabilistic Mendelian randomization. **The American Journal of Human Genetics**, 108(2): 240-256. [doi: 10.1016/j.ajhg.2020.12.006](https://www.sciencedirect.com/science/article/abs/pii/S0002929720304420).

+ Haimiao Chen<sup>$</sup>, Ting Wang<sup>$</sup>, Shuiping Huang<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2021). New novel non-MHC genes were identified for cervical cancer with an integrative approach of transcriptome-wide association analysis. **Journal of Cancer**, 12(3): 840-848. [doi: 10.7150/jca.47918](https://www.jcancer.org/v12p0840.htm).

> ## 2020
+ Lishun Xiao<sup>$</sup>, Zhongshang Yuan<sup>$</sup>, Siyi Jin, Ting Wang and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2020). Multiple-Tissue Integrative Transcriptome-Wide Association Studies Discovered New Genes Associated With Amyotrophic Lateral Sclerosis. **Frontiers in Genetics**, 11:587243. [doi: 10.3389/fgene.2020.587243](https://www.frontiersin.org/articles/10.3389/fgene.2020.587243/full).

+ Xinghao Yu, Ting Wang, Shuiping Huang<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2020). How can gene expression information improve prognostic prediction in TCGA cancers: an empirical comparison study on regularization and mixed-effect survival models. **Frontiers in Genetics**, 11: 920. [doi: 10.3389/fgene.2020.00920](https://www.frontiersin.org/articles/10.3389/fgene.2020.00920/full)

+ Zhongshang Yuan, Huanhuan Zhu, [**Ping Zeng**](https://github.com/biostatpzeng), Sheng Yang, Shiquan Sun, Can Yang, Jin Liu and Xiang Zhou<sup>#</sup> (2020). Testing and controlling for horizontal pleiotropy with the probabilistic Mendelian randomization in transcriptome-wide association studies. **Nature Communications**, 11(1): 3861. [doi: 10.1038/s41467-020-17668-6](https://www.nature.com/articles/s41467-020-17668-6)

+ Xinghao Yu<sup>$</sup>, Zhongshang Yuan<sup>$</sup>, Haojie Lu, Yixin Gao, Haimiao Chen, Zhonghe Shao, Jiaji Yang, Fengjun Guan, Shuiping Huang and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2020). Relationship between birth weight and chronic kidney disease: evidence from systematics review and two-sample Mendelian randomization analysis. **Human Molecular Genetics**, 29(13): 2261-2274. [doi: 10.1093/hmg/ddaa074](https://academic.oup.com/hmg/article-abstract/29/13/2261/5824309?redirectedFrom=fulltext)

+ Xinghao Yu<sup>$</sup>, Ting Wang<sup>$</sup>, Yiming Chen, Ziyuan Shen, Yixin Gao, Lishun Xiao, Junnian Zheng<sup>#</sup> and [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> (2020). Alcohol Drinking and Amyotrophic Lateral Sclerosis: An Instrumental Variable Causal Inference. **Annals of Neurology**, 88(1): 195-198. [doi: 10.1002/ana.25721](https://onlinelibrary.wiley.com/doi/full/10.1002/ana.25721)

> ## 2019
+ [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup>, Ting Wang, Junnian Zheng<sup>#</sup> and Xiang Zhou<sup>#</sup> (2019). Causal association of type 2 diabetes with amyotrophic lateral sclerosis: new evidence from Mendelian randomization using GWAS summary statistics. **BMC Medicine**, 17(1), 225. [doi: 10.1186/s12916-019-1448-9](https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-019-1448-9).

+ Xinghao Yu, Lishun Xiao, [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> and Shuiping Huang<sup>#</sup> (2019). Jackknife Model Averaging Prediction Methods for Complex Phenotypes with Gene Expression Levels by Integrating External Pathway Information. **Computational and Mathematical Methods in Medicine**, 2019: 8. [doi: 10.1155/2019/2807470](https://www.hindawi.com/journals/cmmm/2019/2807470/cta/). [Software](https://github.com/biostatpzeng/JMAP).

+ [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> and Xiang Zhou<sup>#</sup> (2019). Causal effects of blood lipids on amyotrophic lateral sclerosis: a Mendelian randomization study. **Human Molecular Genetics**, 28(4): 688-697. [doi: 10.1093/hmg/ddy384](https://www.ncbi.nlm.nih.gov/pubmed/30445611). 

> ## 2018
+ [**Ping Zeng**](https://github.com/biostatpzeng), Xingjie Hao and Xiang Zhou<sup>#</sup> (2018). Pleiotropic Mapping and Annotation Selection in Genome-wide Association Studies with Penalized Gaussian Mixture Models. **Bioinformatics**, 34(16): 2797-2807. [Doi: 10.1093/bioinformatics/bty204](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty204/4960046?redirectedFrom=fulltext). [PDF](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty204/4960046?redirectedFrom=fulltext), [Software](https://github.com/biostatpzeng/iMAP).

+ Xingjie Hao, [**Ping Zeng**](https://github.com/biostatpzeng), Shujun Zhang and Xiang Zhou<sup>#</sup> (2018). Identifying and exploiting trait-relevant tissues with multiple functional annotations in genome-wide association studies. **PLoS Genetics**, 14(1): e1007186. [Doi: 10.1371/journal.pgen.1007186](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007186). [PDF](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007186), [Software](http://www.xzlab.org/software.html).

> ## 2017
+ [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup>, Ting Wang and Shuiping Huang<sup>#</sup> (2017). Cis-SNPs Set Testing and PrediXcan Analysis for Gene Expression Data using Linear Mixed Models. **Scientific Reports**, 7(1): 15237. [Doi: 10.1038/s41598-017-15055-8](https://www.nature.com/articles/s41598-017-15055-8). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2017SR.pdf), [Software](https://github.com/biostatpzeng/LRT).

+ [**Ping Zeng**](https://github.com/biostatpzeng) and Xiang Zhou<sup>#</sup> (2017). Non-parametric genetic prediction of complex traits with latent Dirichlet process regression models. **Nature Communications**, 8(1): 456. [Doi: 10.1038/s41467-017-00470-2](https://www.nature.com/articles/s41467-017-00470-2?WT.feed_name=subjects_genetics). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2017NC.pdf), [Software](https://github.com/biostatpzeng/DPR) 

+ [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup>, Xiang Zhou and Shuiping Huang<sup>#</sup> (2017). Prediction of gene expression with cis-SNPs using mixed models and regularization methods. **BMC Genomics**, 18(1): 368. [Doi: 10.1186/s12864-017-3759-6](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3759-6). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2017BMCGEMICS.pdf), [Software](https://github.com/genetics-statistics/GEMMA) 

+ Lorin Crawford<sup>#</sup>, [**Ping Zeng**](https://github.com/biostatpzeng), Sayan Mukherjee and Xiang Zhou<sup>#</sup> (2017). Detecting epistasis with the marginal epistasis test in genetic mapping studies of quantitative traits. **PLoS Genetics**, 13(7): e1006869. [Doi: 10.1371/journal.pgen.1006869](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006869). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2017PLOSGENETICS.pdf), [Software](https://github.com/lorinanthony/MAPIT)

> ## Before 2015
+ [**Ping Zeng**](https://github.com/biostatpzeng), Yang Zhao, Chen Qian, Liwei Zhang, Ruyang Zhang, Jianwei Gou, Jin Liu, Liya Liu and Feng Chen<sup>#</sup> (2015). Statistical analysis for Genome-Wide Association Study. **Journal of Biomedical Research**, 29(4): 285-297. [Doi: 10.7555/JBR.29.20140007](http://www.jbr-pub.org/ch/reader/view_abstract.aspx?file_no=JBR150403&flag=1). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2015JBR.pdf)

+ [**Ping Zeng**](https://github.com/biostatpzeng)<sup>#</sup> and Ting Wang (2015). Bootstrap restricted likelihood ratio test for the detection of rare variants. **Current Genomics**, 16(3): 194-202. [Doi: 10.2174/1389202916666150304234203](http://www.eurekaselect.com/129127). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2015CGs.pdf)

+ [**Ping Zeng**](https://github.com/biostatpzeng), Yang Zhao, Hongliang Li, Ting Wang and Feng Chen<sup>#</sup> (2015). Permutation-based variance component test in generalized linear mixed model with application to multilocus genetic association study. **BMC Medical Research Methodology**, 15(37). [Doi: 10.1186/s12874-015-0030-1](https://bmcmedresmethodol.biomedcentral.com/articles/10.1186/s12874-015-0030-1). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2015BMCMM.pdf)

+ [**Ping Zeng**](https://github.com/biostatpzeng), Yang Zhao, Jin Liu, Liya Liu, Liwei Zhang, Ting Wang, Shuiping Huang and Feng Chen<sup>#</sup> (2014). Likelihood ratio tests in rare variant detection for continuous phenotypes. **Annals of Human Genetics**, 78(5): 320-332. [Doi: 10.1111/ahg.12071](http://onlinelibrary.wiley.com/doi/10.1111/ahg.12071/full). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2014AHG.pdf), [Software](https://github.com/biostatpzeng/LRT)

+ [**Ping Zeng**](https://github.com/biostatpzeng), Yang Zhao, Liwei Zhang, Shuiping Huang and Feng Chen<sup>#</sup> (2014). Rare Variants Detection with Kernel Machine Learning Based on Likelihood Ratio Test. **PLoS ONE**, 9(3): e93355. [Doi: 10.1371/journal.pone.0093355](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0093355). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2014PLOSONE.pdf), [Software](https://github.com/biostatpzeng/LRT)

+ [**Ping Zeng**](https://github.com/biostatpzeng), Yongyue Wei, Yang Zhao, Jin Liu, Liya Liu, Ruyang Zhang, Jianwei Gou, Shuiping Huang and Feng Chen<sup>#</sup> (2014). Variable selection approach for zero-inflated count data via adaptive lasso. **Journal of Applied Statistics**, 41(4): 879-894. [Doi:10.1080/02664763.2013.858672](http://www.tandfonline.com/doi/abs/10.1080/02664763.2013.858672). [PDF](https://github.com/biostatpzeng/pingzeng/blob/master/paper/2014ZIP.pdf), [Software](https://github.com/biostatpzeng/ZIP)

